cmake/third_party/LevelZero.cmake

```diff
@@ -3,4 +3,9 @@ FetchContent_Declare(
   URL      https://github.com/oneapi-src/level-zero/archive/v1.0.22.zip
   URL_HASH SHA256=ad9a757c5e07cd44d8e63019e63ba8a8ea19981ea4b7567d3099d3ef80567908
 )
-FetchContent_MakeAvailable(LevelZero)
+FetchContent_GetProperties(LevelZero)
+if(NOT levelzero_POPULATED)
+    FetchContent_Populate(LevelZero)
+    set_property(GLOBAL PROPERTY LEVEL_ZERO_INCLUDE_DIRS ${levelzero_SOURCE_DIR}/include)
```

Comment:
Is this required for the subdir? Or can we add it after? If so, we can still use `MakeAvailable` and add the include dirs after.

Reply:
Thanks, include dirs in rt BUILD file now.